library.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "libusb_stm32",
+	"description": "Lightweight USB Device Stack",
+	"version": "0.1.0",
+	"keywords": "stm32, usb",
+	"authors": {
+		"name": "Dmitry Filimonchuk",
+		"email": "dmitrystu@gmail.com"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/dmitrystu/libusb_stm32.git"
+	},
+	"frameworks": [ "cmsis", "stm32cube" ],
```

Comment:
I don't see a reason why you'd want to restrict frameworks to just these two. At the very least I'd want to be able to use it with `libopencm3` but going with any of them should be possible.

Reply:
after a bit of research it is not clear to me what frameworks are compatible, but i will lift the restriction altogether.